osquery/tables/system/darwin/time_machine_destinations.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <string>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include <osquery/filesystem.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+
+namespace pt = boost::property_tree;
+
+namespace osquery {
+namespace tables {
+
+const std::string kTimeMachinePrefs =
+    "/Library/Preferences/com.apple.TimeMachine.plist";
+const std::string kDestinationKey = "Destinations";
+const std::string kAliasKey = "BackupAlias";
+const std::string kDestinationIdKey = "DestinationID";
+const std::string kBytesUsedKey = "BytesUsed";
+const std::string kBytesAvailableKey = "BytesAvailable";
+const std::string kRVUUID = "RootVolumeUUID";
+const std::string kConsistencyScanDate = "ConsistencyScanDate";
+const std::string kLKEncryptionState = "LastKnownEncryptionState";
+
+QueryData genTimeMachineDestinations(QueryContext& context) {
+  QueryData results;
+  pt::ptree tree;
+  Status s = osquery::parsePlist(kTimeMachinePrefs, tree);
+  if (!s.ok()) {
+    return results;
+  }
+
+  if (tree.count(kDestinationKey)) {
```

Comment:
Would you mind being explicit about your comparison as well? Maybe something like
```
if (tree.count(kDestinationKey) == 0) {
  return QueryData();
}
```